src/main/java/org/jitsi/videobridge/TransportManager.java

```diff
@@ -217,6 +217,36 @@ else if (RawUdpTransportPacketExtension.NAMESPACE.equals(namespace))
 
     /**
      * Sets the values of the properties of a specific
+     * <tt>ColibriConferenceIQ.Endpoint</tt>
+     * to the values of the respective properties of this instance. Thus, the
+     * specified <tt>iq</tt> may be thought of as containing a description of
+     * this instance.
+     *
+     * @param iq the <tt>ColibriConferenceIQ.Endpoint</tt> on which to set the
+     * values of the properties of this instance
+     */
+    public void describe(ColibriConferenceIQ.Endpoint iq)
```

Comment:
Can we avoid the code duplication with public void describe(ColibriConferenceIQ.ChannelBundle iq)?